utils/zerovec/src/ule/error.rs

```diff
@@ -7,42 +7,48 @@ use core::fmt;
 
 /// A generic error type to be used for decoding slices of ULE types
 #[derive(Copy, Clone, Debug)]
-pub enum ULEError {
+pub enum ZeroVecError {
```

Comment:
Nit: If you name this `ZeroVecError`, please move it up to the top level instead of nesting it under `ule`

Reply:
Done. I continued reexporting it from `ule` for the convenience of glob imports